canonical/ubuntu.com
# /desktop/organisations refresh

## Done

- Applied updates as per [doc](https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit) and [design](https://www.figma.com/file/wLOtPeQhAiExhQCwKYhi3H/24.04-U.com---desktop-(rebranding)?node-id=483%3A13939&mode=dev)

Drive by:
- Added latest news partial to revamped pages in the bubble
- Applied me screen layout for resources section across the bubble (Lyubo clarified that this should be the case)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13541.demos.haus/desktop/organisations
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7741
